src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Converters/RequestTelemetryConverter.cs

```diff
@@ -67,12 +76,16 @@ protected override RequestTelemetry CreateTelemetryEntry(LogEvent logEvent, IFor
 
             var requestTelemetry = new RequestTelemetry(sourceName, requestTime, requestDuration, responseStatusCode, isSuccessfulRequest)
             {
-                Id = id,
                 Url = url,
                 Source = source,
                 Context = { Operation = { Name = requestOperationName } }
             };
 
+            if (_options != null)
```

Comment:
Can `_options` ever be null ?

Reply:
Yes, bc this implementation makes sure we support both the 'old' way of injecting the request-specific options, and the 'new' way of options passed-along via the base class.

Reply:
The ID is now set in the `OperationContextConverter` where the correlation assignment is centralized. There, the operation ID is being used as the ID of the request (like proposed in the service-to-service poc).